### ISSUE TITLE
Upgrading to Hasura 2.0 , Track all returns timeout error

### ISSUE THREAD

**Issue by nadendlaprasad** [hasura/graphql-engine#8120]
TrackAll from the schema crashing Hasura and returns timeout error most of the time

**Comment by AndresPrez** [comment]
Same for me. I found and ENV for migration timeout that is not found in the docs. Maybe you also have on for track all?

**Comment by nadendlaprasad** [comment]
can you share the environment config details?

**Comment by AndresPrez** [comment]
@nadendlaprasad turns out the console `Track All` action worked for me but it took about 4 hours locally using `v2.5.0.cli-migrations-v3` docker container in a 16GM RAM 2.2Ghz 6-Core interal i7 Apple Pro

**Comment by nadendlaprasad** [comment]
@AndresPrez agreed. Post 2.0, Track all takes quite long time to process the request

**Comment by dsandip** [comment]
Update: this is being looked at by the team and there should be updates on this in the near future.

**Comment by motya770** [comment]
Would like to receive updates on the issue, important  for our team

**Comment by vijayprasanna13** [comment]
Hey folks! We've added a new experimental version of the Data Tab to handle larger schemas. You can try it out starting from v2.33.0-beta.1 by enabling the feature flag. Please follow this https://github.com/hasura/graphql-engine/discussions/9846 for more updates and do share your early feedback with us!
